index.ts

```diff
@@ -86,6 +88,30 @@ async function main() {
             }
             continue;
           }
+          // If project issue is assigned then close the devpool equivalent
+          if (projectIssue.assignee?.login) {
+            if (devpoolIssue.state === "open") {
+              await octokit.rest.issues.update({
+                owner: DEVPOOL_OWNER_NAME,
+                repo: DEVPOOL_REPO_NAME,
+                issue_number: devpoolIssue.number,
+                state: "closed",
+              });
+              console.log(`Closed (bounty unavailable): ${devpoolIssue.html_url} (${projectIssue.html_url})`);
+            } else {
+              console.log(`Already closed (bounty unavailable): ${devpoolIssue.html_url} (${projectIssue.html_url})`);
+            }
+          } else {
+            if (devpoolIssue.state === "closed" && projectIssue.state === "open") {
+              await octokit.rest.issues.update({
+                owner: DEVPOOL_OWNER_NAME,
+                repo: DEVPOOL_REPO_NAME,
+                issue_number: devpoolIssue.number,
+                state: "open",
+              });
+              console.log(`Opened (bounty available): ${devpoolIssue.html_url} (${projectIssue.html_url})`);
+            } else continue;
```

Comment:
This `continue` should be applied for the [whole block](https://github.com/ubiquity/devpool-directory/blob/add9984a8ee82be14d14ea3171bf518c91bc07f2/index.ts#L91C1-L114C12). So if we close or reopen a devpool issue then we can `continue` otherwise proceed as usual. Simply put, if `octokit.rest.issues.update()` was called the `continue`.